<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlgomoleQtClass</class>
 <widget class="QWidget" name="AlgomoleQtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AlgomoleQt</string>
  </property>
  <widget class="Renderer" name="rendererWidget" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>630</width>
     <height>500</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>630</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>630</width>
     <height>500</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>630</width>
     <height>500</height>
    </size>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QSplitter" name="splitter">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLineEdit" name="idInput">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>PDB ID</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loadButton">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>311</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>123</x>
     <y>200</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1.4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Probe Radius</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Size</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>301</width>
     <height>122</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Space Filling</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Euclidian Distance Transform</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gaussian Approximation</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Meshing</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Marching Cubes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Alpha Wrapping</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>64</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>128</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>256</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Renderer</class>
   <extends>QWidget</extends>
   <header>renderer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="algomoleqt.qrc"/>
 </resources>
 <connections/>
</ui>
